src/readers/morphologySWC.cpp

```diff
@@ -231,9 +231,6 @@ class SWCBuilder
         case 1: {
             return SOMA_SINGLE_POINT;
         }
-        case 2: {
-            return SOMA_UNDEFINED;
-        }
```

Comment:
Why not SOMA_CYLINDERS here ?

Reply:
It is a default soma type. Should I explicitly set it here?

Reply:
I think so yes.

Reply:
done